umh.docs.umh.app/content/en/docs/production-guide/backup_recovery/backup-restore-umh.md

```diff
@@ -73,6 +84,33 @@ sudo $(which kubectl) scale statefulset {{< resource type="statefulset" name="ka
 sudo $(which kubectl) scale statefulset {{< resource type="statefulset" name="mqttbroker" >}} --replicas=0 -n united-manufacturing-hub --kubeconfig /etc/rancher/k3s/k3s.yaml
 ```
 
+### Copy kubeconfig file
+
+1. Show the content of kubeconfig file with the following command and copy the output:
+
+   ```bash
+   sudo cat /etc/rancher/k3s/k3s.yaml
+   ```
+
+   {{% notice tip %}}
+   This guide assumes that the kubeconfig file is located at 
+   `/etc/rancher/k3s/k3s.yaml`. The location depends on your 
+   environment and may differ.
+   {{% /notice %}}
+
+2. Create a `.yaml` file on your local device and paste the content in it.
```

Comment:
It would be better to tell the user to create a new text file, copy the content, then save the file inside the backup folder by renaming it to `k3s.yaml`. This way, when running the backup script, we can directly say that the `-KubeconfigPath` param is set to `.\k3s.yaml`

Reply:
One creates a file with the name before copying and commands are also updated accordingly